usr/share/rear/lib/layout-functions.sh

```diff
@@ -819,6 +819,40 @@ is_disk_a_pv() {
     return 0
 }
 
+# Check whether disk is suitable for being added to layout
+# Can be used to skip obviously unsuitable/broken devices
+# (missing device node, zero size, device can't be opened).
+# Should not be used to skip potential mapping targets before layout restoration
+# - an invalid disk may become valid later, for example if it is a DASD that needs
+# low-level formatting (see 090_include_dasd_code.sh and 360_generate_dasd_format_code.sh),
+# unformatted DASDs show zero size.
+# Returns 0 if the device is ok
+# Returns nonzero code if it should be skipped, and a text describing the error
+# (without the device name) on stdout
```

Comment:
I think the error messages contain the device name
```
echo "$disk is not a block device"
echo "$disk has invalid size $size"
```
Only the `blockdev --getsize64` output
may not contain the device in some cases
but it seems usually the `blockdev --getsize64` output
does contain the device like in your
https://github.com/rear/rear/pull/3047#issue-1890627700
```
Ignoring sdb: blockdev: cannot open /dev/sdb: No medium found
```
or on my system
```
# blockdev --getsize64 /dev/sdb
blockdev: cannot open /dev/sdb: No such file or directory

# blockdev --getsize64 /dev/sda12
blockdev: cannot open /dev/sda12: No such file or directory
```

Reply:
Yes, the message is perfectly well.

All I liked to point out is that the
"(without the device name)" part of the comment
is probably not true.

Reply:
For the fun of it:
I found a blockdev error message that does not show the device:
```
# blockdev --getsize64 /dev/ttyS9
blockdev: ioctl error on BLKGETSIZE64: Input/output error
```
So a message might look like
```
Ignoring sdz: blockdev: ioctl error on BLKGETSIZE64: Input/output error
```
which is still sufficiently well.